### ISSUE TITLE
Поддержка алиасов для объектов и комнат

### ISSUE THREAD

**Issue by dext0r** [dext0r/yandex_smart_home#433]
В 2023.1 появились алиасы (альтернативные названия) для объектов и комнат. Их использования может помочь тем, у кого устройства и комнаты названы по-английски (например у меня 😗)

Предлагается следующий алгоритм для выбора название:
1. По умолчанию используем основное название (поле `Name`/`Название`).
2. Если в основном название нет русских букв, но они есть в одном из алиасов - используем алиас.
3. ~Если есть алиас в формате `Алиса: ***` - принудительно используем `***`.~

Подобный подходит позволит избежать лишних однотипных записей в ямле:
![Screenshot 2023-02-06 at 09 22 43](https://user-images.githubusercontent.com/3103856/216898104-cc28afef-4fe3-4700-b20a-f27340745ee7.png)

**Comment by glebsterx** [comment]
Мне вот кажется, что схему стоит упростить)
Есть алиас -- используем его и все =)

Потому что начнется потом "а как надо 'Алиса: светильник' писать?" с пробелом или нет...

**Comment by dext0r** [comment]
Согласен, пока третий пункт уберём, но если кому-то понадобится - реализуем.

**Comment by glebsterx** [comment]
![image](https://user-images.githubusercontent.com/8779304/216902515-be0e6e2b-db42-45a9-978f-2598843d94e2.png)
я вот называю в ХА свет в комнатах по имени _комнаты_ или "свет в гостиной", а в УДЯ мне надо его прокидывать как "свет" (и он разумеется привязан к комнате), хоть это и неправильно (но мне хочется) 🤷
![image](https://user-images.githubusercontent.com/8779304/216902554-63ce3611-a6fa-4c84-a458-b84f732605e5.png)

А еще допустим у меня есть устройство (физическое) датчик климата, а прокидываю я "датчик температуры" -- я бы всегда его прокидывал как "Температура и влажность", привязывая к нему влажность в конфиге... но это уже за уши притянуто, но...

**Comment by dext0r** [comment]
Несколько дополнений:
1. Сделать глобальную настройку, которая отключит использование алиасов "по дефолту"
2. Если есть алиасы в формате "Алиса: ХХХ" - использовать их (от настройки в пункт 1 не зависит)

Причина - костыли для поддержки падежей во встроенном в HA ассистенте (https://t.me/yandex_smart_home/54730)
